doc/radosgw/notifications.rst

```diff
@@ -162,6 +162,7 @@ Request parameters:
  - if ``ca-location`` is provided, and secure connection is used, the specified CA will be used, instead of the default one, to authenticate the broker
  - user/password may only be provided over HTTPS. If not, topic creation request will be rejected.
  - user/password may only be provided together with ``use-ssl``, if not, the connection to the broker would fail.
+ - user/password may be provided together with ``mechanism`` (default: ``PLAIN``)
```

Comment:
* please provide the valid values for this parameter
* please add a line for "mechanism" after line 125